projects/plugins/jetpack/_inc/client/sharing/publicize.jsx

```diff
@@ -61,14 +61,16 @@ export const Publicize = withModuleSettingsFormHelpers(
 				isOfflineMode = this.props.isOfflineMode;
 
 			const showUpgradeLink =
+				userCanManageModules &&
```

Comment:
I think we are going to get rid of this component soon, given that we now have a dedicated Social admin page.

Reply:
Yeah I thought I'd update it for now